Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM perl:latest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag](https://app.codacy.com/manual/akotlar/bystro/pullRequest?prid=4888220)